sdk/core/Azure.Core/src/RetryOptions.cs

```diff
@@ -22,7 +22,7 @@ internal RetryOptions()
 
         /// <summary>
         /// The delay between retry attempts for a fixed approach or the delay
-        /// on which to base calculations for a backoff-based approach.
+        /// on which to base calculations for a back-off-based approach.
```

Comment:
double hyphen? 👀

Reply:
![image](https://user-images.githubusercontent.com/913445/98290932-5951c600-1f78-11eb-9d3c-e4d13b263403.png)

Reply:
I was thinking of a double guitar 😆

Reply:
I looked it up. You're right. The `backoff` mentioned here is related to this: https://en.wikipedia.org/wiki/Exponential_backoff
```suggestion
        /// on which to base calculations for a backoff-based approach.
```